app/decorators/user_decorator.rb

```diff
@@ -27,4 +27,7 @@ def attendance_count_for_not_required
     not_required_attendance_ids_for_member.count
   end
 
+  def parent_name
+    object.parent_id.present? ? User.find(parent_id).name : '[unknown]'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.